libcudacxx/include/cuda/__device/device_ref.h

```diff
@@ -21,16 +21,16 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/__driver/driver_api.h>
-#include <cuda/std/__cuda/api_wrapper.h>
+#if _CCCL_HAS_CTK()
+#  include <cuda/__driver/driver_api.h>
+#  include <cuda/std/__cuda/api_wrapper.h>
 
-#include <string>
-#include <vector>
+#  include <string>
+#  include <vector>
```

Comment:
Ditto will explode with nvrtc, do we need string, or can we get away with `cuda::std::string_view`?